sample-apps/react-native/dogfood/src/components/VideoWrapper.tsx

```diff
@@ -75,6 +76,21 @@ export const VideoWrapper = ({ children }: PropsWithChildren<{}>) => {
             : undefined,
         },
       });
+
+      _videoClient.on('call.rejected', async (event) => {
+        const callCid = event.call_cid;
+        const callId = callCid.split(':')[1];
+        const rejectedCall = _videoClient?.call('default', callId);
+        await rejectedCall?.getOrCreate();
+
+        const isCalleeBusy =
+          rejectedCall && rejectedCall.isCreatedByMe && event.reason === 'busy';
+
+        if (isCalleeBusy) {
+          Alert.alert('Call rejected because user is busy.');
+        }
+      });
```

Comment:
We leak here. Please unsubscribe this event handler on component unmount.
Also, the `event` carries the `call` data, so no need for `getOrCreate()` as well.